### ISSUE TITLE
Output contains null replacesStatements

### ISSUE THREAD

**Issue by radix0000** [openownership/bods-ingester-gleif#3]
A very small number of (3 out of 13.3 million) statements have a null value for the replacesStatement. This is due to complex series of deletions/replacements GLEIF is performing in these cases not being handled correctly. Given that the whole replacesStatement mechanism now being replaced, work on fixing this has been paused until BODS 0.4 implementation, as it may just work within a better system for handling change over time.

**Comment by kd-ods** [comment]
> complex series of deletions/replacements GLEIF is performing in these cases 

@kathryn-ods - this is the kind of thing we'll be trying to understand with the help of the GLEIF data team, I suppose.

**Comment by kathryn-ods** [comment]
> > complex series of deletions/replacements GLEIF is performing in these cases
> 
> @kathryn-ods - this is the kind of thing we'll be trying to understand with the help of the GLEIF data team, I suppose.

I think in this particular case of it only being 3 records I probably won't raise it as a specific issue but potentially the conversations will give us some context for what is going on?

**Comment by radix0000** [comment]
Yes, when we have time we probably need to work out the exact sequence of "GLEIF events" for each one, and what we think that means. But as I say when we implement BODS 0.4 it may just work, or we can worry about it then if it doesn't.
